### ISSUE TITLE
px-tree slowness

### ISSUE THREAD

**Issue by VI-GE** [PredixDev/px-tree#6]
Hello,

We are currently migrating from jquery tree to px-tree.

We have quite big trees, about 8000 items.

We use a string uuid as the id of a tree node, don't know yet if it might play a factor in this.

The issue is that px-tree is kinda slow when first creating it, I have tried to both set the items all at once, and use addChildren and sometimes even in async/setTimeout without much speed increase.

It seems to also block the animations and requestAnimationFrame requests also lag quite a lot.

This is not a big an issue as it only happens when the tree is first created and I can mitigate this somewhat (progress bar, don't start 3dAnimation until tree is created)

But another issue is that the tree is kinda slow (compared to jquery tree) when opening branches. I am not sure if some lazy loading is done but it seems that reopening previously opened branches seems faster the second time.

If you know some ways to optimize the tree (for example use different types for the id) or if there is a specific way of building it so it doesn't slow down the page, I'd appreciate it.

Also wondering if there is any development work being done or investigation into optimizing px-tree for speed (for big tress)

Thank you
